idoit-install

```diff
@@ -1505,15 +1510,16 @@ EOF
     if [[ -d "/var/log/mysql" ]]; then
         log "Let every user read the logs"
         chmod 755 /var/log/mysql
-        chmod 664 /var/log/mysql/*
+        chmod 664 /var/log/mysql/
     fi
 
-    unitctl "start" "$MARIADB_UNIT"
```

Comment:
Are you sure MariaDB is already running?

Reply:
true, i will revert it